pkg/function/delete_volume_snapshot.go

```diff
@@ -79,15 +79,15 @@ func deleteVolumeSnapshot(ctx context.Context, cli kubernetes.Interface, namespa
 		snapshot, err := provider.SnapshotGet(ctx, pvcInfo.SnapshotID)
 		if err != nil {
 			if strings.Contains(err.Error(), SnapshotDoesNotExistError) {
-				log.Debug().Print("Snapshot already deleted", field.M{"SnapshotID": pvcInfo.SnapshotID})
+				log.WithContext(ctx).Print("Snapshot already deleted", field.M{"SnapshotID": pvcInfo.SnapshotID})
```

Comment:
@ihcsim As per code, I think this shouldn't be part of Debug. We are checking if snapshot exist or not, without Debug level log it will be difficult for user to track what's going wrong.

Reply:
Agreed with your approach :+1:.